lib/deqm_test_kit/measure_availability.rb

```diff
@@ -36,5 +36,27 @@ class MeasureAvailability < Inferno::TestGroup
         output measure_id: resource.entry[0].resource.id
       end
     end
+
+    test do
+      title 'Measure cannot be found returns empty bundle'
+      id 'measure-availability-02'
+      description 'Selected measure is know not to exist on the server and returns an empty bundle'
+      uses_request :measure_search
```

Comment:
I don't actually think we want the `uses_request` pattern here. This will be useful when we want to do something with the response returned for this named request from within another test group without having to duplicate the request. However, this test makes a different request and does not need to rely on the response from the `measure_search` in the previous test.

This actually causes an issue where this test can't be run standalone since it depends on the `measure_search` request having been made by the previous test, even though that isn't actually a dependency of this test